function/.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please do not commit this file here.
There is already one for the whole repo: https://github.com/kyma-incubator/hydroform/blob/master/.gitignore